### PR TITLE
Coord

### PR DESCRIPTION
**Pull request by drneuris**
Coord.py now uses code from
http://home.hiwaay.net/~taylorc/toolbox/geography/geoutm.html
to convert leaflet lat long to DCS internal X-Z, taking into consideration the ellipsoid, false easting/northing and utm zone data from the caucasus config (which is somewhat weird).

Tested different points over the theater seems like there's a consistent (small) error, for now i'm not adding an offset cause it might be because of the map tiles; the rest checks out.

Review, test, like and subscribe, send pizza kthx

### CODE REVIEW

**Comment by bobmoretti** [comment]
Overall looks good to me. Thanks so much for doing this, it's a huge improvement! Would be nice to give names to some of the magic numbers (or at least a few more comments explaining why some of these mysterious numbers work/what function they are serving).

Don't want to throw away the investment of time that you sank into this, but did you have a look to see if any existing python libraries could have done some of these calculations? E.g. https://pypi.org/project/mgrs/ ?

Finally, we probably don't need simple conversion functions to live in a class. I made it that way because originally it had to load those large lookup tables that had to be initialized. But now that you've simplified it, the `coord` function doesn't have any state, so we could later think about just giving it a pure functional interface.

File: coord.py [hunk 1]
```diff
@@ -1,31 +1,422 @@
...
+        zone = 36 #caucasus hardcoded zone 36?? save for later np.floor ((lon + 180.0) / 6) + 1
+        utmxy, zone = self.LatLonToUTMXY(np.deg2rad(lat), np.deg2rad(lon), zone)
+
+        return utmxy[1]-4998115, utmxy[0]-599517
+
+    def xz_to_lat_lon(self, x, z):
+        zone = 36
+        southhemi = False
+
+        utmy = x + 4998115
```
Comment (bobmoretti) [inline coord.py:416]: What do these magic numbers do? A comment or a name for the constant would be helpful for folks maintaining this later.

**Comment by adamdoda** [comment]
As people should start working on the project soon shall we accept this? Magic can be fixed later.

**Comment by bobmoretti** [comment]
> As people should start working on the project soon shall we accept this? Magic can be fixed later.

Agreed. Since this is a major improvement let's get it in. I made a new trello card to do the clean-up so we don't forget.

**Pull request merged**